include/RAJA/pattern/forall.hpp

```diff
@@ -563,28 +607,30 @@ forall_Icount(Res r, Args&&... args)
 namespace detail
 {
 
-template <typename T, typename ExecutionPolicy, typename LoopBody, typename Res>
+template <typename T, typename ExecutionPolicy, typename LoopBody, typename Res, typename ForallParams>
 RAJA_INLINE camp::resources::EventProxy<Res> CallForall::operator()(T const& segment,
                                                                ExecutionPolicy,
                                                                LoopBody body,
```

Comment:
Shouldn't body be forwarded here?

Reply:
Probably, it wasn't before, but this is called inside parallel regions so they should already have their own copies of the body.

Reply:
Ya just seems weird to make another copy.